.github/workflow_scripts/env_setup.sh

```diff
@@ -17,21 +17,23 @@ function setup_build_contrib_env {
 }
 
 function setup_torch_gpu {
-    # Security-patched torch.
-    python3 -m pip install torch==1.13.1+cu117 torchvision==0.14.1+cu117 torchaudio==0.13.1+cu117 --extra-index-url https://download.pytorch.org/whl/cu117
+    PIP_EXTRA_INDEX_URL=https://download.pytorch.org/whl/cu118 reinstall_torch
 }
 
 function setup_torch_cpu {
-    # Security-patched torch
-    python3 -m pip install torch==1.13.1 torchvision==0.14.1 torchaudio==0.13.1 --extra-index-url https://download.pytorch.org/whl/cpu
+    PIP_EXTRA_INDEX_URL=https://download.pytorch.org/whl/cpu reinstall_torch
 }
 
 function setup_torch_gpu_non_linux {
-    pip3 install torch==1.13.1+cu117 torchvision==0.14.1+cu117 --extra-index-url https://download.pytorch.org/whl/cu117
+    setup_torch_gpu
 }
 
 function setup_torch_cpu_non_linux {
-    pip3 install torch==1.13.1 torchvision==0.14.1
+    setup_torch_cpu
+}
+
+function reinstall_torch {
+    pip3 install --force-reinstall torchvision~=0.15.1
```

Comment:
why `pip3` btw? might it be that the autogluon python setup is not running a virtual environment, in which symlinked `pip` will be available? see also my Dockerfile snippet from the earlier comment

Reply:
I think `pip3` is just because of legacy where we thought `pip3` would be correct more often than `pip` (and in most cases it is identical). However, now it is probably the case that `pip3` and `pip` are almost always the same, so it shouldn't matter if it is changed to `pip`